source/java/src/org/lucee/extension/pdf/tag/PDF.java

```diff
@@ -1338,15 +1338,23 @@ private void doActionSetInfo() throws PageException, IOException, DocumentExcept
 
 		PDFStruct doc = toPDFDocument(source, password, null);
 		PdfReader pr = doc.getPdfReader();
-		OutputStream os = null;
+		OutputStream os = new ByteArrayOutputStream();
+
+		if (destination != null && name != null) throw engine.getExceptionUtil().createApplicationException("you cannot use attributes[destination,name] together, must specify one of them");
```

Comment:
You cannot use use both attributes [destination, name] at the same time, only specify one